lib/studies/reply/inbox.dart

```diff
@@ -2,41 +2,51 @@ import 'package:flutter/material.dart';
 import 'package:gallery/layout/adaptive.dart';
 import 'package:gallery/studies/reply/mail_card_preview.dart';
 
+import 'package:provider/provider.dart';
+import 'package:gallery/studies/reply/model/email_store.dart';
+
 class InboxPage extends StatelessWidget {
   const InboxPage({Key key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     final isDesktop = isDisplayDesktop(context);
     final isTablet = isDisplaySmallDesktop(context);
-    final horizontalPadding = isTablet ? 60.0 : isDesktop ? 120.0 : 6.0;
+    final horizontalPadding = isTablet ? 60.0 : isDesktop ? 120.0 : 4.0;
 
-    return Navigator(
-      onGenerateRoute: (settings) {
-        return MaterialPageRoute<void>(builder: (context) {
-          return SafeArea(
-            child: Row(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                Expanded(
-                  child: ListView.separated(
-                    padding: EdgeInsetsDirectional.only(
-                      start: horizontalPadding,
-                      end: horizontalPadding,
-                      top: isDesktop ? 28 : 6,
+    return Consumer<EmailStore>(
+      builder: (context, model, child) {
+        return Navigator(
+          onGenerateRoute: (settings) {
+            return MaterialPageRoute<void>(builder: (context) {
```

Comment:
nit: formatting. More broadly, I think having 3 builders here is a bit much. Consider splitting this into an inbox navigator widget (or moving the navigator logic to InboxPage's parent) and the UI inbox, which would be Consumer  → SafeArea  → Row...

Reply:
I think you're right. I'll update this implementation thanks!